65black/StackOlite
# #160199315 Consume Signup Endpoint

#### What does this PR do?
This PR consumes the `/api/v1/auth/signup` endpoint from [this repo](https://github.com/65black/StackOverflow-lite).

#### How should this be manually tested?
 - Clone this repo and open index.html with Chrome.
 - Navigate to the signup page and try to sign up a new user account.
 - If the user already exists, an error notice will be displayed at the top.
 - If the user doesn't exist, then the registration will be successful and you will be redirected to the login page.

#### Screenshots
![signup](https://user-images.githubusercontent.com/36507972/45615992-04842600-ba66-11e8-81c7-1f8ec0113a32.JPG)

#### What are the relevant pivotal tracker stories?
#160199315
